github_app_for_splunk/default/data/ui/views/security_alert_overview.xml

```diff
@@ -244,7 +251,7 @@
         <option name="totalsRow">false</option>
         <option name="wrap">true</option>
         <format type="color" field="severity">
-          <colorPalette type="map">{"critical":#DC4E41,"high":#F1813F,"moderate":#F8BE34}</colorPalette>
+          <colorPalette type="map">{"critical":#DC4E41,"high":#F1813F,"moderate":#F8BE34, "medium":#F8BE34}</colorPalette>
```

Comment:
Adding medium to the color list.  Dependabot and Code Scanning have different levels of severity.